server/opts.go

```diff
@@ -79,6 +79,8 @@ type ClusterOpts struct {
 
 	// Not exported (used in tests)
 	resolver netResolver
+	// Snapshot of configured TLS options.
+	tlsConfigOpts *TLSConfigOpts
```

Comment:
These are needed in case we cannot find a certificate in the chain, then we load again from the ca file that was configured.